src/components/DetailFine/AlarmRequest_PaymentUpdate/index.tsx

```diff
@@ -0,0 +1,167 @@
+import * as Style from './styles';
+import { SYSTEM } from '@/assets/icons/System';
+import theme from '@/styles/Theme';
+import { CheckDetailFine, SetCheckDetailFine } from '@/components/DetailFine/AlarmRequest_PaymentUpdate/hooks/useCheckDetailFine';
+import { useUpdateDetailStatus } from '@/queries/Detail';
+import { useParams, useSearchParams } from 'react-router-dom';
+import ItemList from './ItemList';
+import { useParticipantList } from '@/queries/Group';
+import { useEffect, useState } from 'react';
+import { Situation } from '@/types/event';
+import { useGroupDetail } from '@/queries/Group';
+
+type Props = {
+  checkDetailFine: CheckDetailFine;
+  setCheckDetailFine: SetCheckDetailFine;
+};
+
+type StatusType = {
+  [key: string]: {
+    title: JSX.Element;
+    subTitle: (situationToChange?: Situation) => JSX.Element;
+  };
+};
+
+const Status: StatusType = {
+  situation_change: {
+    title: <Style.Title>납부 여부 변경하기</Style.Title>,
+    subTitle: (situationToChange?: Situation) => (
+      <Style.SubTitle>
+        선택된 모든 내역을{' '}
+        <span style={{ textDecoration: 'underline', textUnderlineOffset: '2px', color: situationToChange === '완납' ? theme.colors.primary_600 : theme.colors.red_600 }}>
+          {situationToChange}
+        </span>
+        으로 변경하시겠습니까?
+      </Style.SubTitle>
+    ),
+  },
+  alarm_request: {
+    title: <Style.Title>납부 요청하기</Style.Title>,
+    subTitle: () => <Style.SubTitle>선택된 팀원에게 벌금 알림을 보내시겠습니까?</Style.SubTitle>,
+  },
+};
+
+// 해당 Status onSubmit도 추가해서 함께 다루고 싶지만.. 컴포넌트 안에 이 부분을 추가하기 애매해서 일다 ㄴ빼놓았음
+
+// 해당 컴포넌트 네이밍 고민 중..
+// PackageMutate 페이지
+// DetailPackage
+// PaymentStatement
+// alert // change // upDate
+
+// 지금 총무 checkBox 클릭을 어떻게 할건지 얘기를 나눠봐야함
+
+const AlarmRequest_PaymentUpdate = ({ checkDetailFine, setCheckDetailFine }: Props) => {
+  const { groupId } = useParams();
+
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const type = searchParams.get('type');
+
+  const [situationToChange, setSituationToChange] = useState<Situation>('완납');
+
+  const { setInitCheckDetailFine } = setCheckDetailFine;
+
+  const { data: participantData } = useParticipantList(Number(groupId));
+  const participantList = participantData?.content.nicknameList;
+
+  const { data: group } = useGroupDetail(Number(groupId));
+  const isAdmin = group?.content.isAdmin;
+
+  useEffect(() => {
+    closePage();
+  }, []);
+  // 새로고침 시 url이 유지되어있어서 빈 창이 나와있음(새로고침 시 searchParam 지우는 방법이 있는지 찾아볼 예정..)
```

Comment:
저도 코드 살펴보면서 약간 이 부분을 걱정했는데,,
새로고침했을 때 선택한 데이터(state)는 남아있지 않지만 url이 유지되는 부분이 어색한 거 같습니다..!

그냥 recoil로 해버리는 건 어떨까요
```suggestion
  // 새로고침 시 url이 유지되어있어서 빈 창이 나와있음(새로고침 시 searchParam 지우는 방법이 있는지 찾아볼 예정..)
```....??